lab/lab2/js/part3-application.js

```diff
@@ -49,6 +55,18 @@ var getAndParseData = function() {
   criteria happens to be — that's entirely up to you)
 ===================== */
 var plotData = function() {
+    _.forEach(myData, function(x){
+      var marker;
+      if (booleanField && x["UCR Code"] == 600) {
+        marker = L.marker([x.Lat, x.Lng]).bindPopup(x["General Crime Category"]).addTo(map);
```

Comment:
I hadn't requested clarifying comments before this HW was due... but for next time, it'd be helpful to have some comments to help understand what it is you are trying to filter here by so I could get a good idea from the outset without needing to understand all the logic.

But it looks good.